build.gradle

```diff
@@ -45,7 +45,9 @@ allprojects {
       exceptionFormat = 'full'
     }
   }
+  tasks.withType(JavaExec) {
```

Comment:
You should be able to accomplish the same thing by setting an env variable `RUSH_CONFIG_DIR`. There are some more details here: http://www.spinnaker.io/docs/custom-configuration#section-about-notation-used-within-the-yaml-files

I think once this block is removed, and the PR rebased, we're ready to merge.